Templates/Templates.xcodeproj/project.pbxproj

```diff
@@ -376,6 +376,7 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
+				English,
```

Comment:
Let's revert this localization change too 😉

Reply:
Done